version.php

```diff
@@ -28,7 +28,7 @@
 // is a database change in the course of development.  It is used
 // internally to determine when a database upgrade is needed.
 //
-$v_database = 486;
+$v_database = 487;
```

Comment:
would not increment this on patch database updates since not sure what it will do to the admin.php stuff

Reply:
(btw, i can revert this change if needed when i do the patch tonight/tomorrow). i still have one more minor fixup commit to do on my recent changes and I can revert it there. So feel free to just bring this in as is.

Reply:
followed your advice, should be all set now, thanks